.github/workflows/test-linux-cpu.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         fi
 
         # Create Conda Env
-        conda create -yp ci_env python="${PYTHON_VERSION}" numpy libpng jpeg scipy
+        conda create -yp ci_env --quiet python="${PYTHON_VERSION}" numpy libpng jpeg scipy
```

Comment:
this is going to be very frustrating when we have env issues. So... once a week?
😔😔😔😔

Reply:
This only affects the progress bar:

```bash
$ conda create --help | grep -e --quiet
  -q, --quiet           Do not display progress bar.
```

The actual information about the environment, i.e what is installed / upgraded / downgraded is still there: https://github.com/pytorch/vision/actions/runs/4194550278/jobs/7272863693#step:10:168